src/sass/partials/_contacts.scss

```diff
@@ -4,4 +4,275 @@
   align-items: center;
   justify-content: center;
   gap: 30px;
+  @include tablet {
+    display: flex;
```

Comment:
it's not necessary to set display: flex; again in the media if it doesn't change and is always flex